src/Symfony/Component/DependencyInjection/ExpressionLanguageProvider.php

```diff
@@ -38,6 +39,20 @@ public function getFunctions()
             }, function (array $variables, $value) {
                 return $variables['container']->getParameter($value);
             }),
+
+            new ExpressionFunction('env', function ($arg, $default = null) {
+                if (2 > func_num_args()) {
+                    return sprintf('$this->getEnvironmentVariable(%s)', $arg);
+                }
+
+                return sprintf('$this->getEnvironmentVariable(%s, %s)', $arg, $default);
+            }, function (array $variables, $value, $default = null) {
+                if (2 > func_num_args()) {
+                    return $variables['container']->getEnvironmentVariable($value);
```

Comment:
Unless I am quite mistaken, this (and the same three lines down) won't work with `getEnvironmentVariable` declared as protected. I would suggest keeping it public.